neuraxle_tensorflow/observable_classes.py

```diff
@@ -0,0 +1,23 @@
+from abc import ABCMeta, abstractmethod
+from typing import Any
+
+
+class Observable:
+    def __init__(self):
+        self._observers = set()
+
+    def subscribe(self, observer: 'Observer'):
+        self._observers.add(observer)
+
+    def unsubscribe(self, observer: 'Observer'):
+        self._observers.discard(observer)
+
+    def on_next(self, value: Any):
+        for observer in self._observers:
+            observer.on_next(value)
+
+
+class Observer(metaclass=ABCMeta):
```

Comment:
https://news.ycombinator.com/item?id=24241886

Maybe a comment instead. We want our objects to be more concrete or less abstract already.